docs/tutorials/join-dal-baker/run-baker.md

```diff
@@ -19,6 +20,30 @@ Now that you have a DAL node, you can run a baking daemon that can attest to DAL
 Look up how to run programs persistently in the documentation for your operating system.
 You can also refer to [Run a persistent baking node](https://opentezos.com/node-baking/baking/persistent-baker/) on opentezos.com.
 
+   For example, if your operating system uses the `systemd` software suite, your service file might look like this example:
+
+   ```systemd
+   [Unit]
+   Description=Octez baker
+   Wants = network-online.target
```

Comment:
why are certain assignments spaced and others are not? Couldn't we be consistent, for instance, never spaced?

Reply:
Made the spacing consistent.